README.md

```diff
@@ -51,7 +51,22 @@ For example, this request will receive a list of available configuration files a
 GET http://localhost:8080/v1/config?scope=cluster
 ```
 
-Please refer to the swagger API documentation for more information.
+##### For detailed informationa about the API, please generate the swagger API documentation.
+##### Follow the below steps to generate the documentation.
+First run
```

Comment:
I would suggest the wording to be: `Follow the below two steps` instead of
first run 
...
then run.